airbyte-integrations/connectors/source-mailchimp/acceptance-test-config.yml

```diff
@@ -0,0 +1,26 @@
+connector_image: airbyte/source-mailchimp:dev
+tests:
+  spec:
+    - spec_path: "source_mailchimp/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      validate_output_from_all_streams: yes
+#  incremental:
```

Comment:
Why is this commented out?

Reply:
incremental in acceptance tests is currently not supported, so it is disabled in tests.
Reason: test is supposed to accept only {cursor_field: value}, when I need another construction.